tutorials/connect-jamf-pro-to-smallstep.mdx

```diff
@@ -51,7 +51,7 @@ Let’s add the Jamf API credentials you just created to Smallstep.
 
 ### 3. Configure a SCEP Enrollment webhook in Jamf Pro
 
-Smallstep uses Dynamic SCEP certificate enrollment. This webhook allows Jamf to request a dynamic, short-lived challenge string from Smallstep during MDM enrollment. The challenge string is injected into a [SCEP payload](https://developer.apple.com/documentation/devicemanagement/scep) for your clients to authenticate to Smallstep.
+Smallstep uses Dynamic SCEP certificate enrollment. This webhook allows Jamf to request a dynamic, short-lived challenge string from Smallstep during a SCEP certificate request. The challenge string is injected into a [SCEP payload](https://developer.apple.com/documentation/devicemanagement/scep) for your clients to authenticate to Smallstep.
```

Comment:
@hslatman does this look more accurate?

Reply:
Yes, much better. 

Technically, it'll be retrieved _before_ the request, but it's fine in this context.

To make it more clear why SCEP is needed at all, maybe in the first line it needs something along the lines of `Smallstep issues certificates using Dynamic SCEP (a.o.)`. Not thrilled by the "a.o.", so maybe something else, but I think it should somehow indicate that it's only used for specific use cases, and that it's not the only way certificates will be issued.

Reply:
Good point. I did one more round on this, to put SCEP into context a bit better.

Reply:
Looking good 👍